src/software/amazon/ion/IntegerType.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2016 Amazon.com, Inc.  All rights reserved.
+
+package software.amazon.ion;
+
+/**
+ * Indicates the Java type of an Ion <code>int</code> value.
+ */
+public enum IntegerType
+{
+    /**
+     * Fits in the Java <code>int</code> primitive (four bytes).
+     * The value can be retrieved through {@link IonReader#intValue()}
```

Comment:
Let's use text that doesn't imply that we've listed all the relevant APIs.  How about:

> The value can be retrieved through methods like `{@link IonReader#intValue()}` without data loss.